Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,55 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
-        }
-    }
+
+
+			{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4844570)

Reply:
would you mind explaining what needs to be fixed?

Reply:
If you press the tab key it will usually map to 4 space characters or something. This is different on different systems and also programs may handle it differently. Therefor you should replace the "tab symbol" by "whitespace symbols". Visual Studio has an option for this and this should be handled automatically. x: